R/config.R

```diff
@@ -299,11 +298,49 @@ python_environments <- function(env_dirs, required_module = NULL) {
 
 python_config <- function(python, required_module, python_versions, forced = NULL) {
 
+  # add the python bin dir to the PATH (so that any execution of python from
+  # within the interpreter, from a system call, or from within a terminal
+  # hosted within the front end will use the same version of python.
+  #
+  # we do this up-front in python_config as otherwise attempts to discover
+  # and load numpy can fail, especially on Windows
+  # https://github.com/rstudio/reticulate/issues/367
+  python_home <- python_dirs <- dirname(python)
```

Comment:
Did we lose the call to `normalizePath(python_dirs)` here?

Reply:
You're right, I missed that. I'll bring it back.